bundler/helpers/lib/shared_native_helpers.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require "excon"
+
+module SharedNativeHelpers
+  USER_AGENT = "dependabot-core/bundler-helper "\
```

Comment:
I wonder if we should pass the Dependabot::VERSION to the native script as an ENVVAR rather than lose the version string here.

Reply:
yeah good catch 👍